### DISCUSSION TITLE
Myth of Empires

### DISCUSSION THREAD

**Discussion by scarecr0w12** [gameserverapp/Platform#655] [Game request]
With Myth Of Empires re-releasing coming in Feb, would be great to get this added in. It can already be used, and setup, in EA at this time, and its completely playable by anyone in the world at this time as the lawsuit is over. Dedi servers can be setup and accessed, just no support from panel systems at this time

https://steamdb.info/app/1794810/

**Comment by gnarlyvr1** [comment]
Yes please!

**Comment by gnarlyvr** [comment]
Any response to this? I really want to get a MoE server up!

**Reply by scarecr0w12** [reply]
I'd expect not until feb when 1.0 is released given that the current version is still "Beta" and the game is not available on steam yet for purcahse until feb when its rereleased.

**Comment by taychason** [comment]
This shouldn't be too hard to add as it uses alot of the same code as ARK and has an API/chat log system/works well with RCON commands.
